test/src/test/java/org/corfudb/integration/CorfuStoreBrowserIT.java

```diff
@@ -132,6 +132,26 @@ public void browserTest() throws
         Assert.assertEquals(browser.printTableInfo(namespace, tableName), 0);
     }
 
+    /**
+     * Create a table and add data to it using the loadTable command.
+     * @throws IOException
+     */
+    @Test
+    public void loaderTest() throws IOException {
+        final String namespace = "namespace";
+        final String tableName = "table";
+        runSinglePersistentServer(corfuSingleNodeHost,
+                corfuStringNodePort);
+
+        // Start a Corfu runtime
+        runtime = createRuntime(singleNodeEndpoint);
+        final int numItems = 100;
+        final int batchSize = 10;
+
+        CorfuStoreBrowser browser = new CorfuStoreBrowser(runtime);
+        Assert.assertEquals(browser.loadTable(namespace, tableName, numItems, batchSize), batchSize);
```

Comment:
In line with the above comment probably we should explicitly expect numItems/batchSize (in case someone changes numItems)... or use non-divisible numbers for test completeness.

Reply:
the return type is purely for testing - no one using the browser will see it.
thanks